### ISSUE TITLE
Accessibility issue: Documents must have <title> element to aid in navigation on /

### ISSUE THREAD

**Issue by mfranzke** [db-ux-design-system/db-ux-design-system.github.io#33]
## What
An accessibility scan flagged the element `<html lang="en">` on https://design-system.deutschebahn.com/ because documents must have <title> element to aid in navigation. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/document-title?application=playwright.

To fix this, ensure each html document contains a non-empty <title> element.

Specifically:

Fix any of the following:
-   Document does not have a non-empty <title> element

## Acceptance Criteria
- [ ] The specific axe violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific axe violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
